src/main/java/model/RequestType.java

```diff
@@ -0,0 +1,43 @@
+package model;
+
+import java.util.stream.Stream;
+
+public enum RequestType {
+
+    HTML(".html", "text/html;charset=utf-8", "src/main/resources/templates"),
```

Comment:
앗 그런데 "/src/main/resource" 자체도 상수로 빼면 더 깔끔하지 않을까 싶네요. 
```suggestion
    HTML(".html", "text/html;charset=utf-8", BASE_PATH + "/templates"),
```